napari/components/cursor.py

```diff
@@ -34,3 +34,4 @@ class Cursor(EventedModel):
     scaled: bool = True
     size: int = 1
     style: CursorStyle = CursorStyle.STANDARD
+    _view_direction: Optional[Tuple[float, ...]] = None
```

Comment:
We added a view direction property to the Camera, as suggested. We are keeping this here for now since it allows layer mouse callbacks to do 3D interactivity without requiring the viewer to be passed as well.

Reply:
I thought mouse layer callbacks get passed the cursor event, which we've added `view_direction` too already in the earlier PR. I'd like us to bring the cursor event and the cursor more inline with each other, but that can happen later